wrappers/binaries_test.go

```diff
@@ -69,6 +69,9 @@ apps:
  world:
    command: bin/world
    completer: world-completer.sh
+`
+
+const packageHello = packageHelloNoSrv+`
```

Comment:
I think this isn't gofmt'ed since it's missing spaces around the `+`

Reply:
Indeed, nice catch! I fixed it and force pushed (as it was such a small diff)